src/SourceBuild/content/eng/pipelines/templates/steps/create-baseline-update-pr.yml

```diff
@@ -0,0 +1,40 @@
+parameters:
+# The pipeline that is being run
+# Used to determine the correct baseline update tool to run
+# Currently only supports "sdk" and "license"
+- name: pipeline
+  type: string
+
+- name: repo
+  type: string
+
+- name: originalTestResultsPath
```

Comment:
The term TestResults is a little confusing to me as we have a TestResults subfolder in the artifacts directory.  The meaning here is different.  How about simply naming them originalFilesDirectory and updateFilesDirectory?